src/query/xasl.h

```diff
@@ -1073,6 +1076,18 @@ struct xasl_node
   ORDERBY_STATS orderby_stats;
   GROUPBY_STATS groupby_stats;
   XASL_STATS xasl_stats;
+  struct sq_cache
+  {
+    MHT_TABLE *ht;
+    UINT64 size_max;
+    UINT64 size;
+    UINT64 miss_max;
```

Comment:
I think there may be some variables that aren't needed. Because XASL is used a lot, we need to make it as lightweight as possible.